sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -39,21 +39,23 @@ def test_estimate_bandwidth():
     assert 0.9 <= bandwidth <= 1.5
 
 
-def test_estimate_bandwidth_1sample():
+def test_estimate_bandwidth_1sample(global_dtype):
     # Test estimate_bandwidth when n_samples=1 and quantile<1, so that
     # n_neighbors is set to 1.
-    bandwidth = estimate_bandwidth(X, n_samples=1, quantile=0.3)
+    bandwidth = estimate_bandwidth(X.astype(global_dtype), n_samples=1, quantile=0.3)
     assert bandwidth == pytest.approx(0.0, abs=1e-5)
```

Comment:
We should make sure to be using `assert_allclose` in all tests where we introduce the fixture